system/ee/ExpressionEngine/Library/CP/URL.php

```diff
@@ -205,17 +209,29 @@ public function serialize()
             'requested_uri' => $this->requested_uri
         );
 
-        return json_encode($serialize);
+        return serialize($serialize);
```

Comment:
@intoeetive does it make sense to dry this up and only build the array in `__serialize()`.  I think this could just be `return serialize($this->__serialize());`